src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Runtimes/MissingSdkRuntimeDetector.cs

```diff
@@ -48,32 +53,36 @@ protected override Task DisposeCoreAsync(bool initialized)
 
         protected override async Task InitializeCoreAsync(CancellationToken cancellationToken)
         {
-            // Note we don't use the ISafeProjectGuidService here because it is generally *not*
-            // safe to use within IProjectDynamicLoadComponent.LoadAsync.
             _projectGuid = await _project.UnconfiguredProject.GetProjectGuidAsync();
             _missingSetupComponentRegistrationService.RegisterProjectConfiguration(_projectGuid, _project);
-            _ = RegisterSdkRuntimeNeededInProjectAsync(_project);
+            _ = RegisterSdkNetCoreRuntimeNeededInProjectAsync(_project);
```

Comment:
Would an exception here be problematic? If so, consider using `IProjectFaultHandlerService` to monitor the task that's being discarded here, so any errors are not ignored.

```suggestion
            _IProjectFaultHandlerService.Forget(RegisterSdkNetCoreRuntimeNeededInProjectAsync(_project), _project);
```